React/Views/RCTView.m

```diff
@@ -716,6 +716,21 @@ - (NSString *)description
 #endif // TODO(macOS GH#774)
 
 #if TARGET_OS_OSX // [TODO(macOS GH#774)
+- (NSShadow*)shadow {
+  CALayer *layer = self.layer;
+
+  // Workaround AppKit issue with directly manipulating the view layer's shadow.
+  if (layer.shadowColor != nil && layer.shadowOpacity > 0) {
+    NSShadow *shadow = [NSShadow new];
+
+    shadow.shadowColor = [[NSColor colorWithCGColor:layer.shadowColor] colorWithAlphaComponent:layer.shadowOpacity];
+    shadow.shadowOffset = layer.shadowOffset;
+    shadow.shadowBlurRadius = layer.shadowRadius;
+
+    return shadow;
+  }
+  return nil;
+}
```

Comment:
Add an ending `// ]TODO(macOS GH#774)` tag here

Reply:
I think it's already there, after the viewBoundsChanged method